kotlin/usecases/serverless_rds/Readme.md

```diff
@@ -70,592 +70,370 @@ In the **jobs** database, create a table named **Work** that contains the follow
-            val sqlStatement = "update work set status = '$status' where idwork = '$id'"
-            val sqlRequest = ExecuteStatementRequest {
-                secretArn = secretArnVal
-                sql = sqlStatement
-                database = "jobs"
-                resourceArn = resourceArnVal
-            }
-            dataClient.executeStatement(sqlRequest)
-            return id
-
-        } catch (e: SQLException) {
-            e.printStackTrace()
+            sendMsg.send(email, xml)
+        } catch (e: IOException) {
+            e.stackTrace
         }
-        return null
-     }
+        return@runBlocking "Report was sent"
+    }
+}
 
-     // Inject a new submission.
-     suspend fun injestNewSubmission(item: WorkItem): String? {
-        val arc = 0
-        val dataClient = getClient()
-        try {
-
-            val name = item.name
-            val guide = item.guide
-            val description = item.description
-            val status = item.status
-
-            // Generate the work item ID.
-            val uuid = UUID.randomUUID()
-            val workId = uuid.toString()
-
-            // Date conversion.
-            val dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss")
-            val now = LocalDateTime.now()
-            val sDate1 = dtf.format(now)
-            val date1 = SimpleDateFormat("yyyy/MM/dd").parse(sDate1)
-            val sqlDate = Date(date1.time)
-
-            // Inject an item into the system.
-            val sqlStatement = "INSERT INTO work (idwork, username,date,description, guide, status, archive) VALUES('$workId', '$name', '$sqlDate','$description','$guide','$status','$arc');"
-            val sqlRequest = ExecuteStatementRequest {
-                secretArn = secretArnVal
-                sql = sqlStatement
-                database = "jobs"
-                resourceArn = resourceArnVal
-            }
-            dataClient.executeStatement(sqlRequest)
-            return workId
-         } catch (e: SQLException) {
-            e.printStackTrace()
-         } catch (e: ParseException) {
-            e.printStackTrace()
-         }
-        return null
-       }
-      }
- ```
+```    
 
 ### RetrieveItems class
 
-The following Kotlin code represents the **RetrieveItems** class that retrieves data from the **Work** table. Like the **InjectWorkService** class, you must specify ARN values for the secret manager and the Amazon Serverless Aurora database.
+The following Kotlin code represents the **RetrieveItems** class that retrieves data from the **Work** table. Notice that you are required to specify ARN values for Secrets Manager and the Amazon Aurora Serverless database (as discussed in the Creating the resources section). Without both of these values, your code won't work. To use the **RDSDataClient**, you must create an **ExecuteStatementRequest** object and specify both ARN values, the database name, and the SQL statement used to retrieve data from the **Work** table.
```

Comment:
Creating the resources --> Create the resources

Without both of these values, your code won't work. --> You must have both of these values for your code to work.

Reply:
fixed